docker/usr/bin/combine_vcf

```diff
@@ -0,0 +1,180 @@
+                        curr_callset_info["row_idx"] = global_callset_idx
+                        curr_callset_info["idx_in_file"] = local_callset_idx
+                        curr_callset_info["filename"] = vcf_file
+                        callsets_dict[callset_name] = curr_callset_info
+                        local_callset_idx += 1
+                        global_callset_idx += 1
+        json_fn = "callsets_%s.json" % datetime.now().strftime("%y%m%d%H%M")
+        json_fn = os.path.join("/tmp", json_fn)
+        with open(json_fn, 'w') as ofd:
+            json.dump({'callsets' : callsets_dict}, ofd, indent=4, separators=(',', ': '))
+        return json_fn, global_callset_idx
+    
+    @staticmethod
+    def __is_vcf_file_list(fn):
+        with open(fn, 'r') as fd:
+            for line in fd:
+                if line.strip():
+                    return os.path.isfile(line.strip())
+        
+    def check_paths(self):
+        if len(self.vcf_inputfiles) == 1 and self.__is_vcf_file_list(self.vcf_inputfiles[0]):
+            root_path = os.path.dirname(self.vcf_inputfiles[0])
+            with open(self.vcf_inputfiles[0], 'r') as fd:
+                inputs = [line.strip() for line in fd if os.path.isfile(os.path.join(root_path, line.strip()))]
+        else:
+            inputs = [line.strip() for line in self.vcf_inputfiles if os.path.isfile(line)]
+        if inputs:
+            self.vcf_inputfiles = inputs
+            if self.callsets:
+                cs_cfg = json.load(self.callsets)
+                self.num_part_units = 0 - len(cs_cfg["callsets"])
+            else:
+                self.callsets, self.num_part_units = self.__generate_callsets(self.vcf_inputfiles)
+            assert self.num_part_units != 0, "No valid callset/sample found in input files"
+        else:
+            raise RuntimeError("No valid samples input files found")
+
+    def __generate_loader_config(self):
+        loader_cVCF = deepcopy(loader_cfg_t)
+        loader_cVCF["callset_mapping_file"] = self.callsets
+        loader_cVCF['reference_genome'] = self.ref_file
+        loader_cVCF["column_partitions"][0]["vcf_output_filename"] = self.output_file
+        loader_cVCF["vcf_output_filename"] = self.output_file
+        loader_cVCF["size_per_column_partition"] = int(abs(self.num_part_units)) * COL_PARTITION_SIZE_UNIT
+        if self.produce_GT_field:
+            loader_cVCF['produce_GT_field'] = True
+        json_fn = "lc_%s" % datetime.now().strftime("%y%m%d%H%M")
+        json_fn = os.path.join("/tmp", json_fn)
+        with open(json_fn, 'w') as ofd:
+            json.dump(loader_cVCF, ofd)
+        return json_fn
+
+    def combine(self):
+        loader_config = self.__generate_loader_config()
+        the_exec_cmd = ['vcf2tiledb', loader_config]
+        pexec = Popen(the_exec_cmd, shell=False)
+	pexec.communicate()
+        rc = pexec.wait()
+        if rc != 0:
+            print("ERROR: failed to combine vcf files. Error code = ", rc)
```

Comment:
maintain convention as @kgururaj mentioned. INFO messages go to stdout, WARN/ERROR messages go to stderr